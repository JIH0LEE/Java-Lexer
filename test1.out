<OP,->
<INTEGER,0>
<SEMICOLON>
<INTEGER,0>
<ID,abc0>
<SEMICOLON>
<INTEGER,123>
<IF>
<SEMICOLON>
<INTEGER,123>
<ID,if0>
<SEMICOLON>
<CHARACTER,' '>
<SEMICOLON>
<ID,a>
<OP,->
<INTEGER,1>
