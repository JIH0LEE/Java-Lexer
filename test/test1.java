class Human{
    String name ="jiho";
    int age=25;
    char gender='m';

    int getNumber(){
        return age;
    }
}
    