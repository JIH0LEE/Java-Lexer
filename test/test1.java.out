<1,CLASS,class>
<1,ID,Human>
<1,LBRACE,{>
<2,VTYPE,String>
<2,ID,name>
<2,ASSIGN,=>
<2,LITERAL,"jiho">
<2,SEMI,;>
<3,VTYPE,int>
<3,ID,age>
<3,ASSIGN,=>
<3,NUM,25>
<3,SEMI,;>
<4,VTYPE,char>
<4,ID,gender>
<4,ASSIGN,=>
<4,CHARACTER,'m'>
<4,SEMI,;>
<6,VTYPE,int>
<6,ID,getNumber>
<6,LPAREN,(>
<6,RPAREN,)>
<6,LBRACE,{>
<7,RETURN,return>
<7,ID,age>
<7,SEMI,;>
<8,RBRACE,}>
<9,RBRACE,}>
